spring-boot-examples/consumer-app/pom.xml

```diff
@@ -93,6 +93,22 @@
         <groupId>org.springframework.boot</groupId>
         <artifactId>spring-boot-maven-plugin</artifactId>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
```

Comment:
Is this configuration needed in the `consumer-app` and `producer-app` poms if we have `maven.deploy.skip` set in the parent `spring-boot-examples/pom.xml`? The parent configuration should be inherited by all child modules automatically.